packages/theme-chalk/src/date-picker/picker.scss

```diff
@@ -122,7 +122,7 @@
     padding: 0 5px;
     margin: 0;
     font-size: 14px;
-    word-break: keep-all;
+    white-space: nowrap;
```

Comment:
![image](https://github.com/element-plus/element-plus/assets/23251408/76bfd541-ed09-4da2-8ff6-09cfce0f4de0)

This results in a single line display.

Reply:
I think it should be in a single line. Range separator won't be to long.

Reply:
Normally the content of this part will not be very long, but it is inevitable that some of it already has multiple characters. This will be a destructive style change and will squeeze the display of the time on the left and right sides. Most of the competing products are line-changed.

Reply:
Try to use CJK characters, you'll find it also in a single line using word-break: keep-all.
![image](https://github.com/element-plus/element-plus/assets/38392315/5364ae2b-fcb5-431c-acf5-4dd08a7587d3)

Reply:
I think this is a scenario that can be ignored. Only Chinese and Japanese have this situation of sentence segmentation without spaces. The words on the delimiter in these two languages should be short compared to other languages. The current change will cause the time display position to be occupied in the case of multiple words in foreign languages.

Reply:
I updated the style.